vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/main/java/com/vaadin/flow/component/grid/it/GridViewSortingPage.java

```diff
@@ -45,15 +46,20 @@ private void createSorting() {
         grid.addColumn(Person::getFirstName, "firstName").setHeader("Name");
         grid.addColumn(Person::getAge, "age").setHeader("Age");
 
-        grid.addColumn(TemplateRenderer.<Person> of(
-                "<div>[[item.street]], number [[item.number]]<br><small>[[item.postalCode]]</small></div>")
+        Comparator<Person> addressComparator = Comparator
+                .comparing((Person person) -> person.getAddress().getNumber())
+                .thenComparing(
+                        (Person person) -> person.getAddress().getNumber());
+
+        grid.addColumn(LitRenderer.<Person> of(
+                "<div>${item.street}, number ${item.number}<br><small>${item.postalCode}</small></div>")
                 .withProperty("street",
                         person -> person.getAddress().getStreet())
                 .withProperty("number",
                         person -> person.getAddress().getNumber())
                 .withProperty("postalCode",
-                        person -> person.getAddress().getPostalCode()),
-                "street", "number").setHeader("Address");
+                        person -> person.getAddress().getPostalCode()))
+                .setComparator(addressComparator).setHeader("Address");
```

Comment:
This test page was using the deprecated API. Removed the `sortingProperties` parameter and added a comparator to make the column actually sortable (which it wasn't before).